views/partials/_mobile-navigation.njk

```diff
@@ -4,7 +4,14 @@
       <li>
         <div class="app-mobile-nav-subnav-toggler{% if path.startsWith(item.url) %} app-mobile-nav__subnav-toggler--active{% endif %}">
           <a class="govuk-link govuk-link--no-visited-state govuk-link--no-underline app-mobile-nav-subnav-toggler__link js-mobile-nav-subnav-toggler" href="/{{ item.url }}/">
-            {{ item.label }}
+            {% if item.items %}
+              <!-- When JavaScript is enabled, the menu links expand a section below with more content, so we should make them headings -->
+              <h2 class="app-mobile-nav-subnav-toggler__link-heading">{{ item.label }}</h2>
+              <span class="app-mobile-nav-subnav-toggler__link-text">{{ item.label }}</span>
+            {% else %}
+              {{ item.label }}
+            {% endif %}
+
```

Comment:
@36degrees Added a [commit](https://github.com/alphagov/govuk-design-system/pull/1744/commits/c72f3d95e26789676209a67bd0f9c3f0ff7b56f3) to address point 2

We had a discussion on Slack about point 1 which is related to checking `item.items` before making the link a heading. This solution is potentially confusing because we could end up with headings and links styled identically, but visually you cannot differentiate between them. One option would also be to make everything a heading regardless of if it has content beneath it or not, but that could also be potentially confusing to a user navigating by headings if there wasn't any content underneath.

This isn't really an issue yet as all of our links have child items underneath them, so we're adding some logic here to handle a scenario we don't have yet. My preference would be to leave this logic as-is, so if we do add a link in future without child items then we don't end up with a heading with no following content.